plugins/user_mapping/src/interface.h

```diff
@@ -0,0 +1,53 @@
+#ifndef IRODS_HTTP_API_USER_MAPPER_INTERFACE_H
+#define IRODS_HTTP_API_USER_MAPPER_INTERFACE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/// Initializes the user mapping plugin.
+///
+/// \param[in] _args A C-string containing the JSON representing the configuration for the plugin.
+///
+/// \pre \p _args must be a non-null C-string that represents a JSON structure.
+///
+/// \returns A code representing the status of the plugin.
```

Comment:
Is it the status of the plugin or the result of the operation?

Reply:
Didn't specify in the others, it's a code representing the result of the operation.